examples/Receiver_Response.py

```diff
@@ -81,20 +87,18 @@ def get_freqs(thisfile):
         exp_type=data_dir,
     )
     # {{{ fit signal in t domain to complex exponential
-    A, omega, phi, t = symbols("A omega phi t", real=True)
     f = psd.lmfitdata(d)
-    f.functional_form = A * sp.exp(1j * 2 * pi * omega * t + 1j * phi)
+    f.functional_form = A * sp.exp(1j * 2 * pi * nu * t + 1j * phi)
     f.set_guess(
         A=dict(value=5e-3, min=1e-4, max=1),
-        omega=dict(
+        nu=dict(
             value=control["nu_test"][j],
             min=control["nu_test"][j] - 1e4,
             max=control["nu_test"][j] + 1e4,
         ),
         phi=dict(value=0.75, min=-pi, max=pi),
     )
-    f.fit(use_jacobian=False)
-    f.eval()
+    f.fit()
```

Comment:
we don't want eval because we aren't plotting we literally just want the fit parameters that come from the fit (which are a result of f.fit()) we don't need to run eval which runs the fit (using the parameters calculated from fit) along an axis.

Reply:
we do want to use the jacobian because it is more efficient and accurate... somehow... the minimization using the jacobian is still pretty hand wavey in my head and I think its a matrix containing the partial derivatives (with respect to the parameters?) any references or good intro reads appreciated

Reply:
yes, jacobian shows how the shape of the residual varies with changes in parameters -- so it can use the shape to figure out what to change.